department-of-veterans-affairs/next-build
# Update Design System dependencies to latest

# Description
Updates the design system dependencies to their latest versions.

Tested against the existing `/outreach-and-events/events` page and an event detail page.

`component-library` release notes: https://github.com/department-of-veterans-affairs/component-library/releases

## Ticket
Relates to #[19386](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19386).

## Developer Task

```[tasklist]
- [x] PR submitted against the ~`main`~ `Sitewide-integration` branch of `next-build`.
```

## Testing Steps
Regression check Events search and detail pages against what is currently in dev.

## QA steps
What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?

## Screenshots

### Events search page

<details><summary>Desktop</summary>

| Before (dev.va.gov) | After (local) |
| --- | --- |
| <img width="1002" alt="Screenshot 2024-11-14 at 4 51 30 PM" src="https://github.com/user-attachments/assets/4feae485-4ebc-4b16-bb0c-2e334884cf9e"> |<img width="1002" alt="Screenshot 2024-11-14 at 4 51 21 PM" src="https://github.com/user-attachments/assets/1cc0ebcc-4263-4b90-8615-1a2b42acfe0f"> |
| <img width="1001" alt="Screenshot 2024-11-14 at 4 52 02 PM" src="https://github.com/user-attachments/assets/9e57518d-d831-40cb-a372-ae31291d422f"> | <img width="1000" alt="Screenshot 2024-11-14 at 4 51 53 PM" src="https://github.com/user-attachments/assets/a4a2abef-9fca-4405-9cf6-d5ad75f8b338"> |

</details>

<details><summary>Tablet</summary>

| Before (dev.va.gov) | After (local) |
| --- | --- |
| <img width="767" alt="Screenshot 2024-11-14 at 4 52 39 PM" src="https://github.com/user-attachments/assets/87d08eae-570b-4b61-99ae-80f2ec369aff"> |<img width="771" alt="Screenshot 2024-11-14 at 4 52 32 PM" src="https://github.com/user-attachments/assets/1b39c55c-3679-4d11-87cf-0c3ad15dfa7a"> |

</details>

<details><summary>Mobile</summary>

| Before (dev.va.gov) | After (local) |
| --- | --- |
| <img width="400" alt="Screenshot 2024-11-14 at 4 53 01 PM" src="https://github.com/user-attachments/assets/e96d2b1a-76a1-4831-97c3-76163872713c"> |<img width="402" alt="Screenshot 2024-11-14 at 4 52 56 PM" src="https://github.com/user-attachments/assets/c8323a48-bf66-4188-b1a3-ef434f4cde9a"> |

</details>

### Events detail page

Tested `/central-western-massachusetts-health-care/events/73034`

<details><summary>Desktop</summary>

| Before (dev.va.gov) | After (local) |
| --- | --- |
| <img width="1000" alt="Screenshot 2024-11-14 at 4 54 25 PM" src="https://github.com/user-attachments/assets/d7978af7-1abb-4a1c-a42e-cdc855d9c82f"> | <img width="1001" alt="Screenshot 2024-11-14 at 4 54 18 PM" src="https://github.com/user-attachments/assets/db1c77de-6e8c-4d0f-8f67-4b3557894e8b"> |
| <img width="1002" alt="Screenshot 2024-11-14 at 4 55 05 PM" src="https://github.com/user-attachments/assets/2566b1a3-ac95-45bd-b813-9e8061d81594"> | <img width="1000" alt="Screenshot 2024-11-14 at 4 54 46 PM" src="https://github.com/user-attachments/assets/f6bfc3fa-0187-4b74-8073-ff78c98c167f"> |
| <img width="1001" alt="Screenshot 2024-11-14 at 4 55 39 PM" src="https://github.com/user-attachments/assets/a0c3a51c-8d65-4500-965f-d907684bc141"> | <img width="1000" alt="Screenshot 2024-11-14 at 4 55 32 PM" src="https://github.com/user-attachments/assets/3e8bcd4e-83b9-4c8d-8e72-46dd8cc674f1"> |

</details>

<details><summary>Tablet</summary>

| Before (dev.va.gov) | After (local) |
| --- | --- |
| <img width="769" alt="Screenshot 2024-11-14 at 4 54 06 PM" src="https://github.com/user-attachments/assets/70f5f23f-d00d-4e79-8021-bce065499b61"> | <img width="770" alt="Screenshot 2024-11-14 at 4 53 59 PM" src="https://github.com/user-attachments/assets/b20b0715-930c-47cc-b8d9-7e38673befbb"> |

</details>

<details><summary>Mobile</summary>

| Before (dev.va.gov) | After (local) |
| --- | --- |
| <img width="401" alt="Screenshot 2024-11-14 at 4 53 39 PM" src="https://github.com/user-attachments/assets/b3894787-3278-4f9e-82b5-7c29b5c41bde"> | <img width="402" alt="Screenshot 2024-11-14 at 4 53 31 PM" src="https://github.com/user-attachments/assets/acb845bb-2a93-4c64-9593-2f73a87ab49f"> |


</details>

# Reviewer

### Reviewing a PR

This section lists items that need to be checked or updated when making changes to this repository. 

## Standard Checks

```[tasklist]
```

## Merging an Approved Layout

When merging a layout, you must ensure that the content type has been turned on for `next-build` inside the `CMS`. This CMS flag must be turned on for editors to preview their work using the next build preview server.

Resource types (layouts) that have not been approved by design should NOT be pushed to production. Ensure that [slug.tsx](../src/pages/[[...slug]].tsx) does not include your resource type if it is not approved.

The status of layouts should be kept up to date inside [templates.md](./templates.md). This includes QA progress, development progress, etc. A link should be provided for where testing can occur.

## Merging a Non-Approved Layout

Your new resource type should not be included inside [slug.tsx](../src/pages/[[...slug]].tsx). Items added here will go into production once merged into the `main` branch. It is imperative that we do not push anything live that has not been approved.

Ensure that this layout has been added to the [templates.md](./templates.md) file with the current status of the work.
